resources/views/cruds/fields/image.blade.php

```diff
@@ -21,7 +21,7 @@
             {!! Form::file('image', array('class' => 'image form-control')) !!}
         </div>
         <div class="form-group">
-            {!! Form::text('image_url', (!empty($source) ? $source->getImageUrl() : ''), ['placeholder' => __('crud.placeholders.image_url'), 'class' => 'form-control']) !!}
+            {!! Form::text('image_url', $source?->getOriginalImageUrl(), ['placeholder' => __('crud.placeholders.image_url'), 'class' => 'form-control']) !!}
```

Comment:
I lied, this doesn't work and generates an error 🙈  need to test that the source has an image first before calling the function